roles/deploy_cr/README.md

```
@@ -0,0 +1,94 @@
+# Role: deploy_cr
+
+This Ansible role is used to deploy a Kubernetes Custom Resource (CR) to a specified namespace in an OpenShift or Kubernetes cluster.
+
+## Role Variables
+
+| Variable           | Required | Type   | Description                                                                                            
+|--------------------|----------|--------|---------------------------------------------------------------------------------------------------------|                           
+| dc_api_version     | yes      | str    | The API version of the CR (e.g., `hco.kubevirt.io/v1beta1`).                                            |
+| dc_kind            | yes      | str    | The kind of the CR (e.g., `HyperConverged`).                                                            |
+| dc_namespace       | no       | str    | The namespace in which the CR should be created. Defaults to `omit`                                     |
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_only; Actual: leading_and_trailing; Unexpected trailing pipe

(MD055, table-pipe-style)